src/containers/UserProfile/LeftPane.js

```diff
@@ -10,21 +10,32 @@ import AboutUser from "./AboutUser";
 class LeftPane extends Component {
   constructor(props) {
```

Comment:
nome of the file should be LeftPanel

Reply:
I have renamed Files. LeftPane and RightPane to LeftPanel and RightPanel